src/Links/__tests__/Link.test.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { render, fireEvent } from 'test-utils';
+import { spy } from 'sinon';
+import { MemoryRouter } from 'react-router-dom';
+import { strictEqual } from 'assert';
+import Link from '../Link';
+
+describe('Link', function () {
+  let getByText;
+  let clickSpy;
+  beforeEach(function () {
+    clickSpy = spy();
+    ({ getByText } = render(
+      <MemoryRouter>
+        <Link to="/" onClick={clickSpy}>
+          Test Link
+        </Link>
+      </MemoryRouter>
+    ));
+  });
+  afterEach(function () {
+    clickSpy.resetHistory();
+  });
+  it('renders', function () {
+    getByText('Test Link');
+  });
+  it('calls the click handler when clicked', function () {
+    fireEvent.click(getByText('Test Link'));
+    strictEqual(clickSpy.callCount, 1);
+  });
```

Comment:
As noted in the component file, we don't really need an `onClick` prop, so this test case can be removed.

Reply:
Removed through 25eeb3e